core/src/main/resources/reference.conf

```diff
@@ -321,6 +321,18 @@ google {
 # They can be enabled individually in the engine.filesystems stanza and in the config.filesystems stanza of backends
 # There is a default built-in local filesystem that can also be referenced as "local" as well.
 filesystems {
+  blob {
+    class = "cromwell.filesystems.blob.BlobPathBuilderFactory"
+    instance {
+      sas-token = "{SAS TOKEN HERE}"
+      store = "inputs"
+      endpoint = "coaexternalstorage"
+      workspaceId = "{PLACEHOLDER}"
```

Comment:
Looks like we use kebab case in here, can you change this to `workspace-id`?

Reply:
what a perfect term for that format (that I didn't know before). +1 for consistency in naming